### PR TITLE
Update capability, command, tvos doc

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

- Make open-notification false for espresso since it is not available so far
    - https://github.com/appium/appium-espresso-driver/issues/440
- Add `buildToolsVersion` in the capability
- Append tvOS documentation
    - e.g. JS examples

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Madh93 — CHANGES_REQUESTED** [review]
Great job! I think `python` code requires these changes :+1:

File: docs/en/writing-running-appium/ios/ios-tvos.md [hunk 1]
```diff
@@ -38,16 +47,23 @@ element.click
...
 
 ```Python
 # Python
-
 element = driver.find_element_by_accessibility_id('element on the app')
 element.get_attribute('focused')
 element.click()
 driver.query_app_state('test.package.name')
 execute_script('mobile: pressButton', { 'name': 'Home' })
+execute_script('mobile: pressButton', { 'name': 'Up' })
```
[the remark this answers is not in the document]
Comment (KazuCocoa) [reply docs/en/writing-running-appium/ios/ios-tvos.md]: ah. Exactly, thanks!

File: docs/en/writing-running-appium/caps.md [hunk 2]
```diff
@@ -124,6 +124,9 @@ These Capabilities are available only on Android-based drivers (like
 |`disableWindowAnimation`|Set device animation scale zero if the value is `true`. After session is complete, Appium restores the animation scale to it's original value. Defaults to `false` | `true`, `false` |
 |`remoteAppsCacheLimit`| Set the maximum number of remote cached apks (default is 10) which are pushed to the device-under-test's local storage. Caching apks remotely speeds up the execution of sequential test cases, when using the same set of apks, by avoiding the need to be push an apk to the remote file system every time a reinstall is needed. Set this capability to `0` to disable caching. | e.g. `0`, `5`, `20` |
 
+|`buildToolsVersion`| Specify the Android `build-tools` version to be something different than the default, which is to use the most recent version. It is helpful to use particular version while your environment has alpha/beta build tools. | e.g. `'28.0.3'` |
```
Comment (jlipps) [inline docs/en/writing-running-appium/caps.md]: `It is helpful to use a non-default version if your environment uses alpha/beta build tools`

File: docs/en/writing-running-appium/ios/ios-tvos.md [hunk 3]
```diff
@@ -3,32 +3,42 @@
...
+- Be sure tvOS simulator exists in your simulator list if you would like to run your test on tvOS simulators
+    - e.g. `xcrun simctl list | grep "com.apple.CoreSimulator.SimRuntime.tvOS"` shows results
+    - Newer Carthage raises an error like `Could not find any available simulators for tvOS` if tvOS simulators do not exist in tvOS build
+
 ## Limitations
 Gesture commands do not work for tvOS. Some commands such as pasteboard do not work as well.
 
 We can handle `focus` on tvOS by simply pressing keys such as up/down/left/right/home.
-tvOS does actions on the _focused_ element. You can get the value of the `focus` attribute via [Attributes API](http://appium.io/docs/en/commands/element/attributes/attribute/).
+tvOS does actions on the _focused_ element. You can get the value of the `focus` attribute via [Attributes API](http://appium.io/docs/en/commands/element/attributes/attribute/). _Get active element_ API returns the focused element.
```
Comment (jlipps) [inline docs/en/writing-running-appium/ios/ios-tvos.md]: `tvOS performs actions`

File: docs/en/writing-running-appium/ios/ios-tvos.md [hunk 4]
```diff
@@ -3,32 +3,42 @@
...
 
 
 ## Basic Actions
 
+_pressButton_ and get focused element by _get active element_ are basic actions in tvOS.
```
Comment (jlipps) [inline docs/en/writing-running-appium/ios/ios-tvos.md]: pressButton and getting the focused element by get active element are the basic actions for tvOS

File: docs/en/writing-running-appium/ios/ios-tvos.md [hunk 5]
```diff
@@ -3,32 +3,42 @@
...
+Consider adding `wait` methods since tvOS also has animation.
```
Comment (jlipps) [inline docs/en/writing-running-appium/ios/ios-tvos.md]: Consider using

**Review by jlipps — APPROVED** [review]

**Pull request merged**